<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="WINCTRL-L53BSHI" name="com.freecrm.testcases.DulicateCreateUserTst" tests="4" failures="1" timestamp="13 Dec 2019 02:24:40 GMT" time="0.004" errors="0">
  <testcase name="createSuperUser_CopyOfCreateUserTst" time="0.000" classname="com.freecrm.testcases.DulicateCreateUserTst">
    <skipped/>
  </testcase> <!-- createSuperUser_CopyOfCreateUserTst -->
  <testcase name="beforeClassInTest" time="0.004" classname="com.freecrm.testcases.DulicateCreateUserTst">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	valis not matching expected [55] but found [33],
	abc vnvv vnc expected [55] but found [223],
	 expected [ANAD] but found [rVI]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
valis not matching expected [55] but found [33],
abc vnvv vnc expected [55] but found [223],
 expected [ANAD] but found [rVI]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.freecrm.testcases.DulicateCreateUserTst.beforeClassInTest(DulicateCreateUserTst.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- beforeClassInTest -->
  <testcase name="pleaseSkipME" time="0.000" classname="com.freecrm.testcases.DulicateCreateUserTst">
    <skipped/>
  </testcase> <!-- pleaseSkipME -->
  <testcase name="createAdminUser_CopyOfCreateUserTst" time="0.000" classname="com.freecrm.testcases.DulicateCreateUserTst">
    <skipped/>
  </testcase> <!-- createAdminUser_CopyOfCreateUserTst -->
</testsuite> <!-- com.freecrm.testcases.DulicateCreateUserTst -->
